testhelpers/testhelpers.go

```diff
@@ -209,8 +235,13 @@ func CopySingleFileFromImage(dockerCli dockercli.CommonAPIClient, repoName, path
 	return CopySingleFileFromContainer(dockerCli, ctrID, path)
 }
 
-func PushImage(dockerCli dockercli.CommonAPIClient, ref string) error {
-	rc, err := dockerCli.ImagePush(context.Background(), ref, dockertypes.ImagePushOptions{RegistryAuth: "{}"})
+func PushImage(dockerCli dockercli.CommonAPIClient, ref string, registryAuth ...string) error {
```

Comment:
It doesn't look like any of the tests are actually passing a registryAuth

Reply:
Fixed!